### DISCUSSION TITLE
Got Cornered in a multiple "array-in" query! What could be the best workaround?

### DISCUSSION THREAD

**Discussion by sniperadmin** [vuejs/vuefire#1298] [Questions and Help] [answered]
> __Note__  A POC is included below

### Project Stack:
- Nuxt@2.15.8
- @nuxtjs/firebase

### Overview of the issue/question:
In a Firebase document, we have 2 attributes with `<Array>` type
```ts
const Pitch = {
  // ...
  formats: [],
  themes: []
}
```

When trying to filter pitch documents based on both fields, we get trapped in a corner where there is a Firebase caveat in Compound queries `FirebaseError: [code=invalid-argument]: Invalid query. You cannot use more than one 'array-contains' filter`

### Actual POC
The following sandbox illustrates the actual complex component: https://stackblitz.com/edit/nuxt-starter-8dqt1c?file=README.md

### Question
What's the best workaround to get the filtration scenario easier as a development experience?

**Comment by posva** [comment, marked as answer]
I read somewhere in the past that one would just filter in the front. In Vue, you could put that in a computed property

**Reply by sniperadmin** [reply]
It looks like this is the only way to use the client side
